tests/graph/backends/test_implementations.py

```diff
@@ -0,0 +1,257 @@
+"""Tests verifying that the graph protocols are implemented.
+
+Copyright (c) 2023 Pixelgen Technologies AB.
+"""
+
+import igraph as ig
+import networkx as nx
+import pytest
+from pixelator.graph.backends.implementations import (
+    IgraphBasedEdge,
+    IgraphBasedEdgeSequence,
+    IgraphBasedVertex,
+    IgraphBasedVertexClustering,
+    IgraphBasedVertexSequence,
+    NetworkxBasedEdge,
+    NetworkxBasedEdgeSequence,
+    NetworkxBasedVertex,
+    NetworkxBasedVertexClustering,
+    NetworkxBasedVertexSequence,
+)
+
+
+@pytest.fixture
+def ig_graph():
+    yield ig.Graph()
```

Comment:
Since it allows us to mix setup/teardown in the fixtures. For these fixtures it makes no difference, since there is no teardown. I just tend to do this on all fixtures.